### DISCUSSION TITLE
GUI

### DISCUSSION THREAD

**Discussion by alecdibble** [sandarshsridhar/wiz-spotify-connected#37] [General]
I had some ideas for introducing a web-based GUI. If I create a PR, are you open to reviewing and merging it into this project?

**Comment by sandarshsridhar** [comment]
@alecdibble, Absolutely!! That would be great!

**Reply by sandarshsridhar** [reply]
Please mention in this thread what framework you're going to use, and an overview on the overall design before you create a PR. Thanks.
